src/Exceptions/InvalidConfiguration.php

```diff
@@ -1,13 +0,0 @@
-<?php
-
-namespace Spatie\Backup\Exceptions;
-
-use Exception;
-
-class InvalidConfiguration extends Exception
```

Comment:
I tried to find where this was used. I can't find it. It seems unused to me. Or is it somehow used somewhere?

Reply:
I'll remove this one later.